src-tauri/src/util.rs

```diff
@@ -175,15 +175,18 @@ pub fn check_ea_app_or_origin_running() -> bool {
     x
 }
 
-/// Checks if Northstar process is running
+/// Checks if Northstar process is running (ignores dedicated server)
 pub fn check_northstar_running() -> bool {
-    let s = sysinfo::System::new_all();
-    let x = s
-        .processes_by_name("NorthstarLauncher.exe")
-        .next()
-        .is_some()
-        || s.processes_by_name("Titanfall2.exe").next().is_some();
-    x
+    let mut s = sysinfo::System::new_all();
+    s.refresh_all();
+    for process in s.processes().values() {
+        if (process.name().ends_with("Titanfall2.exe") || process.name().ends_with("Northstar.exe"))
```

Comment:
Related to https://github.com/R2NorthstarTools/FlightCore/pull/562#issuecomment-1723204223 I think, process name was changed in this PR while it shouldn't have been modified:
```suggestion
        if (process.name().ends_with("Titanfall2.exe") || process.name().ends_with("NorthstarLauncher.exe"))
```